packages/core/src/types/IViewport.ts

```diff
@@ -97,9 +105,17 @@ export interface ViewReference {
    *
    * The naming of this particular attribute matches the DICOM SR naming for the
    * referenced image, as well as historical naming in CS3D.
+   *
+   * For range/selection, this must be the starting range referenced image id
    */
   referencedImageId?: string;
 
+  /**
+   * An internal URI version of hte referencedImageId, used for performance
+   * while checking the referenced image id.
+   */
+  referencedImageUri?: string;
```

Comment:
we have `FrameOfReferenceUID` above, so is this `referencedImageURI` ? 

I don't see anywhere else in the code we have `referencedImageUri` but i see 16 occurences of `referencedImageURI`

Reply:
Updated the docs to explain it - there is no relationship to FrameOfReferenceUID

Reply:
we don't have `referencedImageUri` at all in the codebase and have 16 `referencedImageURI` already, that is my point